server/checks/Specification.js

```diff
@@ -9,6 +9,8 @@ const DEFAULT_REQUIRED_LENGTH = 8
 const ISSUE_PATTERN = /^(?:[-\w]+\/[-\w]+)?#\d+$/
```

Comment:
The `DEFAULT_REQUIRED_LENGTH` we should pull from the default config instead of hardcoding, e.g. `const DEFAULT_REQUIRED_LENGTH = getIn(nconf.get('ZAPPR_DEFAULT_CONFIG'), ['specification', 'title', 'minimum-length', 'length'])`, also because it can be different for title and body. But for now we can keep it.